pkg/k8s/network_policy_test.go

```diff
@@ -1640,10 +1640,10 @@ func (s *K8sSuite) TestGetPolicyLabelsv1(c *C) {
 		c.Assert(lbls, NotNil)
 		c.Assert(len(lbls), Equals, 4, Commentf(
 			"Incorrect number of labels: Expected Name, UID, Namespace and DerivedFrom labels."))
```

Comment:
You need to fix the comment

Reply:
Changed the order in the error message, too.